subsys/bluetooth/audio/pbp.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/types.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/pbp.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/check.h>
+
+LOG_MODULE_REGISTER(bt_pbp, CONFIG_BT_PBP_LOG_LEVEL);
+
+/* PBA Service UUID + Public Broadcast Announcement features + Public_Info metadata of length 1 */
+#define MIN_PBA_SIZE			5
```

Comment:
Shouldn't this be
```suggestion
/* PBA Service UUID + Public Broadcast Announcement features + Public_Info metadata of length 1 */
#define MIN_PBA_SIZE (BT_UUID_SIZE_16 + 1 + 1)
```

i.e. the minimum size is 4, and not 5? 
Couldn't this value be used in the check below
```c
CHECKIF(pba_data_buf->size < meta_len + 3) {
```
?